gulpfile.js

```diff
@@ -0,0 +1,52 @@
+const { src, watch, series, dest } = require("gulp");
+const sass = require("gulp-sass");
+const del = require("del");
+
+// directory for building SCSS, and bundles
+const buildDir = "./public/scss/libs/protocol/";
+const finalDir = "./public/css/";
+
+function cleanCompiledCssDirectory() {
+    return del([
+        "./public/css/*",
+        "!./public/css/legacy/**",
+    ]);
+}
+
+function resetCssDirectories() {    
+    return del([
+        buildDir,
+        "./public/css/*",
+        "!./public/css/legacy/**",
```

Comment:
micro-optimization: 
```
const compiledCssDirectories = [
        "./public/css/*",
        "!./public/css/legacy/**",
    ]
```

And then re-use it in both `cleanCompiledCssDirectory` and `resetCssDirectories`.

Reply:
On the `resetCssDirectories()`, I also have another item to add, so I ended up `.concat`'ing the two arrays. LMK if its too messy.

Reply:
Resolved this with @pdehaan